overlays/hpack.nix

```diff
@@ -0,0 +1,12 @@
+self: super:
+{
+  haskell = super.haskell // {
+    packages = super.haskell.packages // {
+      ghc865 = super.haskell.packages.ghc865.override {
+        overrides = hself: hsuper: {
+          hpack = self.haskell-nix.tool "ghc865" "hpack" "0.34.2";
```

Comment:
The repetition of `"ghc865"` is sloppy.  We could DRY that up and iterate over all the package sets.

I didn't use `packageOverrides`, because though I can get the GHC version from `hself`, I can't figure out how to get the package name without string munging.

If I anticipated this overlay lasting a long time, I'd be more ashamed of it.